assets/javascripts/discourse/connectors/topic-footer-main-buttons-before-create/assign-button.js.es6

```diff
@@ -3,7 +3,7 @@ import { ajax } from 'discourse/lib/ajax';
 
 export default {
   shouldRender(args, component) {
-    const needsButton = component.currentUser && component.currentUser.get('staff');
+    const needsButton = component.currentUser && component.currentUser.get('staff') && component.siteSettings.assign_enabled;
```

Comment:
I would love to know why! Did it not come through in `Discourse.Site.currentProp('disabled_plugins')`?